src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,14 +1,36 @@
 package mate.academy.spring.dto.request;
 
+import mate.academy.spring.service.Email;
+import mate.academy.spring.service.RepeatedPassword;
+
+@RepeatedPassword
 public class UserRequestDto {
+    @Email
     private String email;
     private String password;
+    private String twinPassword;
```

Comment:
```suggestion
    private String repeatPassword;
```
"twinPassword" - sounds like "doubled password"

Reply:
well. ok